docker/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM node:8-alpine
+
+RUN apk add --update --no-cache \
+                            git \
+                            libzmq \
+                            zeromq-dev \
+                            python \
+                            make \
+                            g++
+
+WORKDIR /insight
+
+# Copy dashcore-node
+RUN git clone --branch master --single-branch --depth 1 https://github.com/dashevo/dashcore-node.git .
```

Comment:
Can we pin this to a hash please?

Reply:
we don't use this, but it's in the "official" repo